### PR TITLE
create inital readme file

### PR DESCRIPTION
**Pull request by Asmaathabet**
relates #22
- create initial read me file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+
+- people who can't reach to the right kind of therapies
+    
+
+## **Solution** 
+
+- make app to facilitate user reaching to the appropirate therpay and connect with therapist At the right time and at with less costs .
+
+
+## **User Journey**
```
Comment (AbdallahAmmar96) [inline README.md]: Take the user journey from the issues

File: README.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
 # Shrinc
```
Comment (AbdallahAmmar96) [inline README.md]: add the team members

File: README.md [hunk 3]
```diff
@@ -1,3 +1,44 @@
...
+## **Table of Contents** 
+* [Problem](#problem)
+* [Solution](#solution)
+* [User Journey](#user-journey)
+* [User Story](#user-story)
+* [Resources](#resources)
+
+## **Problem** 
+
+- people who are busy  and can't find therapist quickly and easily
```
Comment (AbdallahAmmar96) [inline README.md]:
```suggestion
- people who are busy and can't find a therapist quickly and easily
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,44 @@
...
+  
+- people who didn't have enough money to ask therapists and get contact with them
+
+- people who can't reach to the right kind of therapies
+    
+
+## **Solution** 
+
+- make app to facilitate user reaching to the appropirate therpay and connect with therapist At the right time and at with less costs .
```
Comment (AbdallahAmmar96) [inline README.md]: Without a point

File: README.md [hunk 5]
```diff
@@ -1,3 +1,44 @@
...
+* [Resources](#resources)
+
+## **Problem** 
+
+- people who are busy  and can't find therapist quickly and easily
+  
+- people who didn't have enough money to ask therapists and get contact with them
+
+- people who can't reach to the right kind of therapies
+
```
Comment (AbdallahAmmar96) [inline README.md]: delete this line

File: README.md [hunk 6]
```diff
@@ -1,3 +1,44 @@
...
+* [User Journey](#user-journey)
+* [User Story](#user-story)
+* [Resources](#resources)
+
+## **Problem** 
+
+- people who are busy  and can't find therapist quickly and easily
+  
+- people who didn't have enough money to ask therapists and get contact with them
+
```
Comment (AbdallahAmmar96) [inline README.md]: people who don't know the type of therapy

File: README.md [hunk 7]
```diff
@@ -1,3 +1,44 @@
...
+- As a user, I can visit the website and find out more about the service
+- As a user, I can complete a questionnaire that will guide me towards the most suitable therapies 
+- As a user, I can view therapist's profiles
+- As a user, I can choose to search (filter) for therapists by various categories (possible to search instead of doing questionnaire)
+    - location (need to think about implementing this properly)
+    - cost (from list of price ranges)
+    - speciality (from a list)
+    - skype/remote possibilty (yes or no)
+- As a user, I can learn more about the different types of therapy available
+
```
Comment (AbdallahAmmar96) [inline README.md]: delete this line

File: README.md (hunk 1 above)
Comment (fadeomar) [inline README.md]: add the user Journey that I added

Comment (Asmaathabet) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Asmaathabet) [reply README.md]: Done

**Pull request merged**